version/version.go

```diff
@@ -22,14 +22,15 @@ import (
 
 const (
 	// OperatorVersionString is the version string of this operator binary.
-	OperatorVersionString = "1.3.0"
+	OperatorVersionString = "1.3.1"
 )
 
 var (
 	// SupportedVersions is a list of chart versions supported by this version of the operator.
 	// It must be synced with the versions.yaml file.
 	SupportedVersions = []string{
 		"1.3.0",
+		"1.3.1",
```

Comment:
We also now have 1.3.2 and .3
Separately, we have to figure out how to keep this up to date with releases....

Reply:
Fixed.

Reply:
Added.